docker-compose.yml

```diff
@@ -6,6 +6,7 @@ services:
         BUILD_ENV: dev
         POETRY_INSTALL_ARGS: ''
       target: dev
+    env_file: .env
```

Comment:
Question: What's the reason for this change?

Reply:
I was having trouble getting the Docker container to pick up the `.env` file I created locally (containing the test AWS bucket creds). It only seemed to work after adding this change, then trashing and rebuilding the container.